cmd/xgql/main.go

```diff
@@ -306,6 +313,17 @@ func main() { //nolint:gocyclo
 			ReadHeaderTimeout: 5 * time.Second,
 			ErrorLog:          stdlog.New(io.Discard, "", 0),
 		}
+		if len(*clientCABundle) > 0 {
+			p, err := getCertPool(*clientCABundle)
+			if err != nil {
+				kingpin.FatalIfError(err, "cannot initialize the client CA pool from the specified bundle %s", *clientCABundle)
+			}
+			srv.TLSConfig = &tls.Config{
+				ClientAuth: tls.VerifyClientCertIfGiven,
+				ClientCAs:  p,
+				MinVersion: tls.VersionTLS12,
```

Comment:
Added this to make the `gosec` linter happy. The library documentation says TLS 1.2 is currently the default minimum. So, this should not be a breaking change if I'm not missing anything...